underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -185,6 +191,9 @@ public boolean delete(String path, boolean recursive) throws IOException {
   @Override
   public boolean exists(String path) throws IOException {
     String newPath = stripPrefixIfPresent(path);
+    if (newPath.endsWith("_temporary")) {
```

Comment:
same here, adding a comment would be very helpful for readability and maintainability